src/pages/index.md

```diff
@@ -21,7 +21,7 @@ description: Write Clarity smart contracts, build apps, and starting mining with
 ## Start mining
 
 [@page-reference | grid]
-| /start-mining/mainnet, /start-mining/testnet, /understand-stacks/running-testnet-node, /understand-stacks/running-mainnet-node
+| /start-mining/mainnet, /start-mining/testnet, /understand-stacks/running-mainnet-node, /understand-stacks/running-testnet-node, /understand-stacks/running-regtest-node
```

Comment:
another part where we should add the regtest link: https://blockstack-docs-git-regtest-blockstack.vercel.app/understand-stacks/overview#try-it-out